pkg/upgrade/upgrade.go

```diff
@@ -65,6 +94,66 @@ func TryUpgrade(cli *clientv3.Client) error {
 
 // upgradeToVer1 does upgrade operations from Ver0 to Ver1.
 // in fact, this do nothing now, and just for demonstration.
-func upgradeToVer1(cli *clientv3.Client) error {
+func upgradeToVer1(cli *clientv3.Client, uctx Context) error {
+	return nil
+}
+
+// upgradeToVer2 does upgrade operations from Ver1 to Ver2 (v2.0.0-rc3) to upgrade syncer checkpoint schema
+func upgradeToVer2(cli *clientv3.Client, uctx Context) error {
+	upgradeTaskName := "upgradeToVer2"
+	logger := log.L().WithFields(zap.String("task", upgradeTaskName))
```

Comment:
`task` has a special meaning in DM (and you also use it in L118 below, in order to match `ExecuteSQLWithIgnoreError`?). How about giving it another name like `job`/`op`...

Reply:
I think internal upgrading is semantic suitable for a task, though other tasks are user specifed in task config. for example, user may see many "task name" running in grafana when upgrade, but in fact that's a internal upgrading not many task running.

and `updateSyncerCheckpoint` of `v1dbschema` uses task name from user's task config, maybe I could change that

Reply:
> and updateSyncerCheckpoint of v1dbschema uses task name from user's task config, maybe I could change that

OK. 👍

Reply:
https://github.com/pingcap/dm/pull/988/commits/e618c7682835a60c6475ae9945cf7ef5b0ed651b